bindings/electron/src/lib.rs

```diff
@@ -10,6 +12,22 @@ lazy_static::lazy_static! {
         Mutex::new(tokio::runtime::Runtime::new().expect("Cannot start tokio runtime for libparsec"));
 }
 
+static SENTRY: OnceCell<ClientInitGuard> = OnceCell::new();
+
+fn init_sentry() {
+    if let Ok(sentry_url) = std::env::var("SENTRY_URL") {
+        SENTRY.get_or_init(|| {
+            sentry::init((
+                sentry_url,
+                ClientOptions {
+                    release: sentry::release_name!(),
+                    ..Default::default()
+                },
+            ))
+        });
+    }
```

Comment:
Should we add a log indicating that sentry was or wasn't configured ?

Reply:
@touilleMan